bpo-35181: Correct importlib documentation for some module attributes (#15190)
python/cpython: Doc/library/importlib.rst

            package. This attribute is not set on modules.

        - :attr:`__package__`
            The fully-qualified name of the package under which the module was

>             loaded as a submodule (or the empty string for top-level modules).

Reviewer: I would recommend removing "(or the empty string for top-level modules)". This recommendation is based upon the [devguide](https://devguide.python.org/documenting/#economy-of-expression). I don't think it is particularly necessary to mention this in multiple areas. Instead, it could be mentioned in a single note below the description for `__package__`.
```suggestion
            loaded as a submodule.
```

Reviewer 2: I think the comment in the original PR is valuable here though.  This section is talking about the legacy API so having to "fast forward" in the documentation (since no link to the below section is provided) is a bit inconvenient.  The original text of the PR seems fine to me.

   (``__loader__``)

>    The :class:`Loader <importlib.abc.Loader>` that should be used when loading

Reviewer: Rather than using the `class` role, `term` should be used to link to the glossary defintion.
```suggestion
   The :term:`Loader` that should be used when loading
```

Reviewer 2: @aeros Thanks, agreed!

>    the module.  Finders should always set this.

Reviewer: The `term` role should be used to link to the glossary definition for `finder`.   
```suggestion
   the module.  :term:`Finders <finder>` should always set this.
```

Reviewer 2: Agreed, thanks.

   (``__package__``)

   (Read-only) The fully-qualified name of the package under which the module

>    should be loaded as a submodule (or the empty string for top-level modules).

Reviewer: ```suggestion
   should be loaded as a submodule.
```

Reviewer 2: Similarly, I think it's fine to mention the empty string here too.
